roles/edpm_users/tasks/create_users_and_groups.yml

```diff
@@ -14,11 +14,16 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Create host user and group
+- name: Create host user and group, if not exists
   become: true
   tags:
     - edpm_users
   block:
+    - name: Detect when user exists and cannot be created
+      ansible.builtin.getent:
```

Comment:
I can see the user created[1] when the rpm is installed. What scenario do we expect the user not to be created during install and we need to do all this? 

[1] [1] https://pkgs.devel.redhat.com/git/rpms/openvswitch/tree/openvswitch.spec?h=rhel-9.0.0#n329

Reply:
when user exists, like comes from RPM spec, it will not be created by this role.

Reply:
do we need extra conditions and tasks here? For example, we can revert logic and add rescue block to user role instead, to make it bypassing creation of a user w/o error if there is that 'usermod' message in the error?